src/util/articleRenderers.ts

```diff
@@ -54,7 +52,10 @@ export const renderArticle = async (json: any, articleId: string, dump: Dump, ca
             await articleDetailXId.set(_articleId, _articleDetail);
         }
 
-        const html = renderMCSArticle(json, dump, _articleId, _articleDetail);
+        if(capabilities.mobileRestApiAvailable || !capabilities.desktopRestApiAvailable){
```

Comment:
Why testing `!capabilities.desktopRestApiAvailable` ? Does not make sense to me.